adaptive_shield/README.md

```diff
@@ -0,0 +1,18 @@
+## Overview
+SaaS Apps like Office 365, Slack, Zoom, and Salesforce are vital to a business's day-to-day operations but  often introduce some new security challenges. Adaptive Shield's SaaS Security Posture Management solution (SSPM) features proactive, deep, continuous and automated monitoring and management of all business critical SaaS applications. By integrating with all SaaS apps within minutes, Adaptive Shield enables security teams to gain complete control over their SaaS apps, allowing them to harden their security policies and reduce risks.
+
+With the Adaptive Shield integration, you can track and monitor SaaS posture alerts, such as configuration drifts, integration failures and security checks degradations, all within the events stream in Datadog.
+
+## Setup
+
+1. On the integration tile in Datadog, click on **"Connect Accounts"**, or go to [Adaptive Shield][1]
+2. Click on **"Settings"**
+3. Go to **Alerts** -> **"Add New Alerts Channel"** -> Select **"Datadog"**
+4. Provide alias name
+5. Complete the process by using OAuth
```

Comment:
this step is unclear. What does the user click/where do they go to complete the process with OAuth within Adaptive Shield?

Does the user still have to set up steps 3-5 after authorizing?

Reply:
Hi, after filling the alias name, the user should click on "OAuth" in Adaptive Shield to complete the OAuth process